tracer.go

```diff
@@ -205,6 +213,10 @@ func (t *Tracer) startSpanWithOptions(
 		if hasParent && parent.isDebugIDContainerOnly() {
 			ctx.flags |= (flagSampled | flagDebug)
 			samplerTags = []Tag{{key: JaegerDebugHeader, value: parent.debugID}}
+			textMapPropagator, ok := t.injectors[opentracing.TextMap].(*textMapPropagator)
+			if ok {
+				samplerTags = []Tag{{key: textMapPropagator.headerKeys.JaegerDebugHeader, value: parent.debugID}}
```

Comment:
@yurishkuro so you're saying that [this](https://github.com/uber/jaeger-client-go/blob/master/tracer.go#L207) pre-existing code is no longer necessary?